src/pages/TransactionDuplicate/DuplicateTransactionItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {useOnyx} from 'react-native-onyx';
+import * as ReportActionsUtils from '@libs/ReportActionsUtils';
+import ReportActionItem from '@pages/home/report/ReportActionItem';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Transaction} from '@src/types/onyx';
+
+type DuplicateTransactionItemProps = {
+    transaction: OnyxEntry<Transaction>;
+    index: number;
+};
+
+function DuplicateTransactionItem(props: DuplicateTransactionItemProps) {
+    const [report] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${props.transaction?.reportID}`);
+    const [reportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report?.reportID}`);
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/non-nullable-type-assertion-style
```

Comment:
```suggestion
    
    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/non-nullable-type-assertion-style
```

Reply:
Also you need to need to commit this one! Our style usually mentions to leave an empty line before comments